src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/TableParsers/CommitmentsTableParser.cs

```diff
@@ -47,7 +47,7 @@ private static CommitmentsTableColumnStructure ParseCommitmentsTableStructure(Ta
                     case "agreed price":
                         structure.PriceIndex = c;
                         break;
-                    case "commitment Id":
+                    case "commitment id":
                         structure.CommitmentIdIndex = c;
                         break;
                     case "version id":
```

Comment:
line 172 - effectiveTo is set to null in an if condition but it has already been initialised above. Consider invertinf the if statement:

```
  if ((structure.EffectiveToIndex <= -1 || !string.IsNullOrEmpty(row[structure.EffectiveToIndex]))
                && structure.EffectiveToIndex > -1 && !TryParseNullableDateTime(row[structure.EffectiveToIndex], out effectiveTo))
            {
                throw new ArgumentException($"'{row[structure.EffectiveToIndex]}' is not a valid effective to");
            }
```

Reply:
Line 208 - unneeded , after pathwayCode